tests/test_github.py

```diff
@@ -501,4 +515,4 @@ def test_failed_change_at_head(self):
         self.assertTrue(C.is_merged)
         self.assertNotIn('merge', A.labels)
         self.assertNotIn('merge', B.labels)
-        self.assertNotIn('merge', C.labels)
\ No newline at end of file
+        self.assertNotIn('merge', C.labels)
```

Comment:
Vim is doing that. Whoever edited this file before wasn't using a proper editor. :)